inc/napa/v8-helpers/json.h

```diff
@@ -27,9 +27,15 @@ namespace v8_helpers {
         }
 
         /// <summary> JSON.parse </summary>
+        #if (V8_MAJOR_VERSION == 6 && V8_MINOR_VERSION >= 6 || V8_MAJOR_VERSION > 6)
```

Comment:
The change of `v8::JSON::Parse()` happens in v8 >= 6.x, not 6.6

Reply:
udpated v8_major_version >= 6